### ISSUE TITLE
Invalid database connection with 8.8.0

### ISSUE THREAD

**Issue by cedricve** [directus/app#2904]
• Version of Directus: 
Directus 8.8.0
• Server, OS & Database Details
<img width="784" alt="Screenshot 2020-07-04 at 09 27 34" src="https://user-images.githubusercontent.com/1546779/86507456-9d6d6800-bdd8-11ea-906a-95c0ee62bc87.png">
• Error Messages
    {"error":{"code":19,"message":"Invalid database connection"}}
• Steps to Reproduce
<img width="1163" alt="Screenshot 2020-07-04 at 09 29 34" src="https://user-images.githubusercontent.com/1546779/86507492-e45b5d80-bdd8-11ea-812e-20e5c59860ae.png">
• Relevant Info, Screenshots, Schema & Logs
I can connect to an existing database from a separate script, as mentioned here.

<?php

$link = mysqli_connect("mysql", "root", "xxx", "xxx");

if (!$link) {
echo "Error: Unable to connect to MySQL." . PHP_EOL;
echo "Debugging errno: " . mysqli_connect_errno() . PHP_EOL;
echo "Debugging error: " . mysqli_connect_error() . PHP_EOL;
exit;
}

echo "Success: A proper connection to MySQL was made! The my_db database is great." . PHP_EOL;
echo "Host information: " . mysqli_get_host_info($link) . PHP_EOL;

mysqli_close($link);

?>

<img width="798" alt="Screenshot 2020-07-04 at 09 31 06" src="https://user-images.githubusercontent.com/1546779/86507525-208ebe00-bdd9-11ea-9c9a-36eaab0a60c0.png">

**Comment by rijkvanzanten** [comment]
Are you sure you're using the same credentials? Are you running your little test script on the same server as Directus? Directus does the same thing internally, so if your custom script is able to reach it, so is directus.

**Comment by cedricve** [comment]
Yes credentials are correct, and I was able to finish the installation by using the command line **bin/directus**.
Something is going wrong in the PHP code, at the installation process.

Probably I'm not the only one suffering from this issue.

**Comment by behrmann97** [comment]
Hello, I have the exact same problem, everythng seems to be fine and the credentials work both on php and manually opening mysql. I'm not sure what to do.

**Comment by andersonbraz** [comment]
Check Here
https://github.com/directus/directus/issues/2496

**Comment by git00000** [comment]
I had the same problem. 
Changing database host from 'localhost' to '127.0.0.1' worked for me.
